internal/cache/cache.go

```diff
@@ -0,0 +1,57 @@
+package cache
+
+import (
+	"github.com/Code-Hex/go-generics-cache/policy/lru"
```

Comment:
It is used a simple, but extremely fast caching library. According to my tests, it is faster than BigCache 5 times.